src/textual/css/_style_properties.py

```diff
@@ -909,7 +910,17 @@ def __set__(self, obj: StylesBase, style_flags: Style | str | None):
                             self.name, word, context="inline"
                         ),
                     )
-            style = Style.parse(style_flags)
+            try:
+                style = Style.parse(style_flags)
```

Comment:
I'm confused about what happens if set the style to `none`.

ie. `my_widget.styles.text_style == "none"`

AFAICT this will throw an exception. It *should* set the style to `Style.NULL`

Reply:
If you set the style to `none`, it sets the style to `none` 🤷  Don't take my word for it, I just added a new test that ensures this is the case.

`Style.parse` can handle the string `"none"` (that's the first thing that `Style.parse` does – it checks if the argument is the string `"none"`.

What fails is if `style_flags` is a space-separated list of styles, one of which is `none`. When that's the case, `Style.parse` doesn't realise we are setting a null style so it proceeds to parsing the full space-separated list and then fails to acknowledge the `none` that is present.

Reply:
Ah. I had forgotten that `Styles.parse` has an explicit for the word "none"